# HG changeset patch
# User David Sehr <sehr@google.com>
# Parent 9336f08363ecf37390135d8738abffcfecf276a6
Removed obsolete ELF start intrinsics and front end support.

diff -r 9336f08363ec lib/Target/X86/X86NaClRewritePass.cpp
--- a/lib/Target/X86/X86NaClRewritePass.cpp	Tue Aug 09 16:42:04 2011 -0700
+++ b/lib/Target/X86/X86NaClRewritePass.cpp	Wed Aug 10 11:37:27 2011 -0700
@@ -463,9 +463,7 @@
     return true;
   }
 
-  if (Opc == X86::NACL_ELF_START32 ||
-      Opc == X86::NACL_ELF_START64 ||
-      Opc == X86::NACL_LONGJ32 ||
+  if (Opc == X86::NACL_LONGJ32 ||
       Opc == X86::NACL_LONGJ64) {
     // The expansions for these intrinsics already handle control SFI.
     return false;
